src/composables/ViewRoutingSlip/useRoutingSlipInfo.ts

```diff
@@ -77,9 +77,9 @@ export default function useRoutingSlipInfo (props) {
 
   const showAddress = computed(() => {
     return (
-      (isRefundProcess(currentStatus?.value) && canRequestRefund.value) ||
-        false
-    )
+      isRefundProcess(currentStatus?.value) ||
+      currentStatus?.value?.code === SlipStatus.ACTIVE
```

Comment:
It already shows when it's in `IsRefundProcess` which is ACTIVE

Reply:
IC `Hide chequeAdvice field when it's empty` ok should be good

Reply:
<img width="395" alt="image" src="https://github.com/bcgov/fas-ui/assets/16507223/6d26e318-bc98-450e-90c4-509b8e05e4c3">
<img width="462" alt="image" src="https://github.com/bcgov/fas-ui/assets/16507223/31a428e8-33d9-413c-99ea-84464855a647">

I looked at the IsRefundProcess function, ACTIVE not in, IsRefundProcess referenced on few other spots, so I think better not add ACTIVE to it.

Reply:
Yup, should be fine, I didn't see the condition you added above